website/docs/concepts/scopes.mdx

```diff
@@ -0,0 +1,331 @@
+---
+title: Scopes
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Scoping in Riverpod is a very powerful feature, but like all powerful features, it should be used wisely and intentionally.
```

Comment:
I would add that it is primarily used for performance optimization and that for any other use case, chances are you should avoid it (in favor of family/autoDispose)